lib/worker.js

```diff
@@ -18,8 +18,9 @@ const process = require('process');
 const { createDirectories, cleanupDirectories } = require('./prepare');
 const { getSource, putRendition } = require('./storage');
 const Rendition = require('./rendition');
+const utils = require('./utils/utils');
```

Comment:
kind of confusing to have a utils file in the `utils` folder

Reply:
Better naming suggestion?

Reply:
My 2 cents:
- Put https://github.com/adobe/asset-compute-sdk/blob/NUI-867/lib/utils/utils.js#L52 in its own file, eg ùtils/console`
- THen either move the 2 remaining functions in `util/timer` or rename the file to `duration`, dealer's choice.

Reply:
They were so small that I thought a separate file is overkill .... see the comment at the top:

```
// place for small utilities. larger parts should move into their own module.
```

Reply:
I agree it is so small that it's overkill. it's fine as is for now